lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -331,7 +331,7 @@ def upgrade_package host, package_name
         #
         # @return nil
         def add_system32_hosts_entry(host, opts = {})
```

Comment:
A meta question, why not use puppet to manage the hosts file, e.g. `puppet resource host ...`?